Sources/BaggageContext/BaggageContext.swift

```diff
@@ -0,0 +1,59 @@
+/// A `BaggageContext` is a heterogenous storage type with value semantics for keyed values in a type-safe
+/// fashion. Its keys are uniquely identified via `BaggageContextKey`s. These keys also dictate the type of
```

Comment:
```suggestion
/// fashion. Its values are uniquely identified via `BaggageContextKey`s. These keys also dictate the type of
```

I guess? Keys are the identifying bits :)